scripts/TabScreen.gd

```diff
@@ -0,0 +1,9 @@
+extends Control
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+func _process(delta):
+	# if TAB is pressed, display the TabScreen
+	visible = Input.is_action_pressed("ui_focus_next")
```

Comment:
Curiosity: where are these event-names defined?  Are they all Godot-level names or do you get to define them?

Reply:
The event names like `"ui_focus_next"` can be defined by a user in the Input Map section of the Project Settings. There's an example about 1/3 down [this tutorial page](https://docs.godotengine.org/en/stable/getting_started/first_2d_game/03.coding_the_player.html).